packages/common/src/core/schemas/internal/metrics/InitiativeUiSchemaMetrics.ts

```diff
@@ -195,6 +195,18 @@ export const buildUiSchema = async (
                   labelKey: "shared.fields.metrics.sourceLink.message.required",
                   allowShowBeforeInteract: true,
                 },
+                {
+                  type: "ERROR",
+                  keyword: "format",
+                  icon: true,
+                  labelKey:
+                    "shared.fields.metrics.sourceLink.message.formatError",
```

Comment:
I might suggest making this `labelKey` something like `shared.errors.urlFormat` - we use this same error message for many fields, and it might be good to have standardized `shared.errors` translations in the entity editor. 

Note: this wouldn't apply for the `StatCardUiSchema`

Reply:
Valid. Pushed up :)